joint_state_topic_hardware_interface/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 cmake_minimum_required(VERSION 3.16)
 project(joint_state_topic_hardware_interface CXX)
 
-if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-  add_compile_options(-Werror -Wall -Wextra -Wpedantic -Wshadow
-    -Wsign-conversion -Winit-self -Wredundant-decls)
+find_package(ros2_control_cmake REQUIRED)
```

Comment:
this is not released for humble now. Do you want to release this package for humble, too?